src/Components/UserInfo/UserInfo.tsx

```diff
@@ -1,15 +1,21 @@
-import React, { useContext } from 'react';
+import React from 'react';
 import './UserInfo.scss';
-import { SetUserContext, UserContext } from '../../ContextProviders/UserProvider';
 
 
 const UserInfo: React.FC = () => {
-  const userData  = useContext(UserContext)
-  const setUserData  = useContext(SetUserContext)
```

Comment:
Glad we removed context. This will make it simpler. We will need to pass props into this component so we have data to render. Can we go ahead and set up props on this component so that we can pass something in from Dashboard?

Reply:
Agreed

Reply:
Just pushed up the updates you requested. Nice catch!